server/src/main/java/org/apache/iotdb/db/mpp/plan/scheduler/StandaloneScheduler.java

```diff
@@ -123,6 +124,13 @@ public void start() {
         LOGGER.info("{} state tracker starts", getLogHeader());
         break;
       case WRITE:
+        // reject non-query operations when system is read-only
+        if (IoTDBDescriptor.getInstance().getConfig().isReadOnly()) {
+          TSStatus failedStatus = new TSStatus(TSStatusCode.NODE_READ_ONLY.getStatusCode());
```

Comment:
Why do you use `TSStatusCode.NODE_READ_ONLY` rather than `TSStatusCode.READ_ONLY_SYSTEM_ERROR` ?

Reply:
What's the difference between NODE_READ_ONLY and READ_ONLY_SYSTEM_ERROR? What scenarios should I use them?

Reply:
NODE_READ_ONLY was used in old cluster only, which means one node is read-only. 
READ_ONLY_SYSTEM_ERROR is used in the current standalone version.

Reply:
Fixed.